### PR TITLE
Named closure parameters

### PR DESCRIPTION
**Pull request by nafetswirth**

### CODE REVIEW

**Review by alexcristici — APPROVED** [review]

File: README.markdown [hunk 1]
```diff
@@ -533,21 +533,50 @@ UIView.animate(withDuration: 1.0, animations: {
...
+```swift
+attendeeList.sort {
+  return $0 > $1
+}
+```
+
+If possible avoid more than one line if the expression is easy to evaluate
+**Preferred**:
+```swift
+attendeeList.sort { $0 > $1 }
```
Comment (Czajnikowski) [inline README.markdown:574]: How about using closures/free funtions? Should we prefer them in a simple cases like this (I mean `sort(by: >)` for example)?

File: README.markdown [hunk 2]
```diff
@@ -533,21 +533,50 @@ UIView.animate(withDuration: 1.0, animations: {
...
+```
+**Not Preferred**:
+```swift
+attendeeList.sort { a,b in
+  return a > b
+}
+```
```
Comment (Czajnikowski) [inline README.markdown]: I think the amount of statements is a better/more strict metric. How about "For closures with more than one statement"?

**Review by Czajnikowski — CHANGES_REQUESTED** [review]
I suggest using statements vs lines in closures 🙂

File: README.markdown (hunk 1 above)
Comment (nafetswirth) [reply README.markdown:574]: Yeah maybe this is a bad example. Don't think it makes a difference in this rule tho

File: README.markdown (hunk 2 above)
Comment (nafetswirth) [reply README.markdown]: @alexcristici are you okay with this?

Comment (nafetswirth) [reply README.markdown]: @alexcristici still waiting for your thoughts on this one

Comment (alexcristici) [reply README.markdown]: Agree, sorry didn’t see it.

**Comment by nafetswirth** [comment]
@alexcristici This was my legacy, you can't just merge that 😛

**Pull request merged**